examples/openapi/README.md

```diff
@@ -32,5 +32,54 @@ curl -X GET http://localhost:8080/openapi
 #Output: [lengthy OpenAPI document]
 ```
 
-The output describes not only then endpoints in `GreetService` as described in
-the static file but also an endpoint contributed by the `SimpleAPIModelReader`.
+The output describes the endpoints in `GreetService` as described in
+the static file.
+
+## Customizing OpenAPI Behavior
+Helidon automatically discovers that OpenAPI is on the classpath, initializes it from configuration, and 
+adds OpenAPI support to the webserver.
+
+You can customize the behavior of OpenAPI in two ways.
+
+### Customizing via Configuration
+The `application.yaml` in the example project already contains a section for `server`. 
+To customize the OpenAPI behavior, add a `features` subsection to `server` as shown below.
+```yaml
+server:
+  port: 8080
+  host: 0.0.0.0
+  features:
+    openapi:
+      web-context: myopenapi
+```
+The configuration changes the endpoint that serves the OpenAPI document from
+the default `/openapi` to `/myopenapi`. 
+
+The [documentation for OpenAPI configuration](https://helidon.io/docs/v4/config/io_helidon_openapi_OpenApiFeature) shows all the settings available for customization.
+
+### Customization via Code
+The `Main#setup` method in this example project initializes the Helidon webserver using the following code:
```

Comment:
Add "By default" to the start of the sentence.

"By default, the `Main#setup` method in this example..."

Reply:
I like 'As written'